scripts/k8s_deploy_rook.sh

```diff
@@ -6,7 +6,9 @@
 # `helm upgrade --namespace rook-ceph rook-ceph rook-release/rook-ceph --version v0.9.0-174.g3b14e51`
 
 HELM_ROOK_CHART_REPO="${HELM_ROOK_CHART_REPO:-https://charts.rook.io/release}"
-HELM_ROOK_CHART_VERSION="${HELM_ROOK_CHART_VERSION:-v1.1.1}"
+
+# Always use the latest unless any specific reason for a particular release
+#HELM_ROOK_CHART_VERSION="${HELM_ROOK_CHART_VERSION:-v1.1.1}"
```

Comment:
This is bad practice, we want to always peg to a specific version.

Can you update this with whatever the current version is?